.github/workflows/release_portal.yml

```diff
@@ -1,16 +1,12 @@
 name: Publish portal
 
 on:
-    push:
-        branches:
-            - main
+    workflow_call:
         paths:
-            # Bare publiser etter at Lerna har versjonert
-            - "portal/package.json"
+            - "portal/**"
```

Comment:
Jeg finner ikke noen konkrete eksempler i dokumentasjonen på hvorvidt dette kommer til å fungere, men [denne siden](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_call) skriver:

> When a workflow is triggered with the workflow_call event, the event payload in the called workflow is the same event payload from the calling workflow.

Om det faktisk betyr at `paths` kan brukes er ikke helt klart. De sikter nok heller til `github.event.sender` og så videre. Men, vi får prøve. Alternativt får vi bare ta kostnaden ved å bygge og publisere portalen for hver merge til main.

Reply:
> Alternativt får vi bare ta kostnaden ved å bygge og publisere portalen for hver merge til main.

Tenker jo egentlig at det er riktigere å gjøre uansett? De _aller fleste_ endringer i repoet vil jo indirekte påvirke portalen ved at det man endrer enten brukes eller dokumenteres eller begge deler. Så får det heller være at det ikke er alle endringer som fører til en merkbar forskjell.

Reply:
Da gjør vi heller det. Keep it simple. Oppdaterer.